### PR TITLE
[ADD] saas_portal.server: add local_host and local_port for server-si…

### PR DESCRIPTION
**Pull request by iledarn**
…de requests

### CODE REVIEW

**Comment by iledarn** [comment]
@yelizariev

**Comment by iledarn** [comment]
@yelizariev , 
https://github.com/yelizariev/odoo-saas-tools/issues/259

File: saas_portal/models/saas_portal.py [hunk 1]
```diff
@@ -75,18 +76,19 @@ def _request(self, **kwargs):
...
             'token_type': 'Bearer',
             'access_token': access_token,
             'expires_in': 3600,
         })
-        url = '{scheme}://{saas_server}:{port}{path}?{params}'.format(scheme=scheme, saas_server=self.local_host or self.name, port=self.local_port or port,
-                                                                      path=path, params=werkzeug.url_encode(params))
-        return url
+        url = '{scheme}://{saas_server}:{port}{path}'.format(scheme=scheme, saas_server=self.local_host or self.name, port=port, path=path)
+        req = requests.Request('GET', url, data=params, headers={'host': self.name})
+        prepped = requests.Session().prepare_request(req)
```
Comment (yelizariev) [inline saas_portal/models/saas_portal.py]: prepped = req.prepare()

File: saas_portal/models/saas_portal.py [hunk 2]
```diff
@@ -267,13 +269,13 @@ def _create_new_database(self, dbname=None, client_id=None, partner_id=None, use
...
         scope = ['userinfo', 'force_login', 'trial', 'skiptheuse']
-        url = server._request_server(path='/saas_server/new_database',
+        prepped = server._request_server(path='/saas_server/new_database',
                               scheme=scheme,
                               port=port,
                               state=state,
                               client_id=client_id,
                               scope=scope,)[0]
-        res = requests.get(url, verify=(self.server_id.request_scheme == 'https' and self.server_id.verify_ssl))
+        res = requests.Session().send(prepped, verify=(self.server_id.request_scheme == 'https' and self.server_id.verify_ssl))
```
Comment (yelizariev) [inline saas_portal/models/saas_portal.py]: update `_request_server` and make it 

```
 return req, req_kwargs
```

and add verify value to req_kwargs

**Comment by iledarn** [comment]
@yelizariev

**Pull request merged**